packages/charts/src/renderers/xy-renderer.ts

```diff
@@ -18,5 +29,17 @@ export abstract class XYRenderer<TA extends IAccessors> extends Renderer<TA> {
             y: scales.y.convert(dataSeries.accessors.data.y(point, index, dataSeries.data, dataSeries)),
         };
     }
+    public getDataPointIndex(series: IDataSeries<TA>, values: { [p: string]: any }, scales: Scales): number {
+        if (isUndefined(values.x)) {
```

Comment:
And, I think the LineRenderer version of this method can be removed too with one minor addition here:

```suggestion
        if (!this.config.interactive || isUndefined(values.x)) {
```